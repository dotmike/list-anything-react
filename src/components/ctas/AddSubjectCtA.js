import React from "react";
import { Card, Icon } from "semantic-ui-react";
import { Link } from "react-router-dom";

const AddSubjectCtA = () => (
  <Card centered>
    <Card.Content textAlign="center">
      <Card.Header>Add new subject</Card.Header>
      <Link to="/subjects/new">
        <Icon name="plus circle" size="massive" />
      </Link>
    </Card.Content>
  </Card>
);

export default AddSubjectCtA;
