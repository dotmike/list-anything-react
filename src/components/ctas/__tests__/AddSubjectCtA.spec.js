import React from "react";

import AddSubjectCtA from "../AddSubjectCtA";

describe("AddSubjectCtA", () => {
  it("renders correctly", () => {
    const wrapper = shallowWithContext(<AddSubjectCtA />);
    expect(wrapper).toMatchSnapshot();
  });
});
