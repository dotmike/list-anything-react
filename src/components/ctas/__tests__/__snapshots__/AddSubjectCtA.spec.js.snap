// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AddSubjectCtA renders correctly 1`] = `
<Card
  centered={true}
>
  <CardContent
    textAlign="center"
  >
    <CardHeader>
      Add new subject
    </CardHeader>
    <Link
      replace={false}
      to="/subjects/new"
    >
      <Icon
        as="i"
        name="plus circle"
        size="massive"
      />
    </Link>
  </CardContent>
</Card>
`;
